### PR TITLE
Fixes #9645 - Added rubocop and travis support

### PR DESCRIPTION
**Pull request by daviddavis**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+language: ruby
+
+rvm:
+  - 2.1.0
```
Comment (domcleal) [inline .travis.yml:4]: You should support down to version 1.8.7 with Hammer, soon to be 1.9.3+.

Comment (domcleal) [inline .travis.yml:4]: Oh, I suppose this is only rubocop, no actual tests :(

File: Rakefile [hunk 2]
```diff
@@ -8,7 +19,9 @@ namespace :gettext do
     require 'gettext/tools'
 
     domain = HammerCLIForemanBootdisk::I18n::LocaleDomain.new
```
Comment (domcleal) [inline Rakefile]: Bootdisk?

**Comment by domcleal** [comment]
I've enabled Travis CI on this repository, try pushing again to this branch to trigger a build.

File: .travis.yml (hunk 1 above)
Comment (daviddavis) [inline .travis.yml:4]: Yea :/

File: Rakefile (hunk 2 above)
Comment (daviddavis) [inline Rakefile]: Yea, in case we need to generate gettext in HammerCLIForemanBootdisk

**Comment by daviddavis** [comment]
@domcleal everything is updated if you want to review again.

**Comment by domcleal** [comment]
@daviddavis oh sorry, please don't consider that a review.  More like fly-by comments!

**Comment by daviddavis** [comment]
@mbacovsky mind reviewing this?

**Comment by parthaa** [comment]
ack.

**Pull request merged**